src/pages/SelectPage.js

```diff
@@ -1,15 +1,23 @@
 import React from "react";
-import "./SelectPage.css";
+import styled from "@emotion/styled";
 
 import Characters from "../components/Characters";
 import Button from "../components/Button";
 import { Link } from "react-router-dom";
 
+const Headline = styled.header`
```

Comment:
a `h` element would probably be more appropriate here, since it's just text that will be displayed. A `header` is good if there are more child elements